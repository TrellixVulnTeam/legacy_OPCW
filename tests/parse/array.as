# Declare some random array expressions using the literal notation.
[];
[1, 2, 3, 4923.312,];
[1, 2, 3];
[false, true];
