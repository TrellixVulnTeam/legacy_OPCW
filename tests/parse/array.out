ModuleDecl <?> _
  Nodes <?> 
    ArrayExpr <?>
      Elements <?> 
    ArrayExpr <?>
      Elements <?> 
        IntegerExpr <?> 1 (10)
        IntegerExpr <?> 2 (10)
        IntegerExpr <?> 3 (10)
        FloatExpr <?> 4923.312
    ArrayExpr <?>
      Elements <?> 
        IntegerExpr <?> 1 (10)
        IntegerExpr <?> 2 (10)
        IntegerExpr <?> 3 (10)
    ArrayExpr <?>
      Elements <?> 
        BooleanExpr <?> false
        BooleanExpr <?> true
